main.go

```diff
@@ -29,15 +29,15 @@ func main() {
 	if !(patchMode == "replace" || patchMode == "add") {
 		panic("Invalid patch mode specified, expecting either 'replace' or 'add'")
 	}
-	if !(mode == "add" || mode == "patch") {
-		panic("Invalid mode specified, expecting 'add' or 'patch'")
+	if !(mode == "add" || mode == "patch" || mode == "both") {
```

Comment:
we might have delete workload in the future as well, `both` wouldn't help then. Maybe be explicit and say `add_and_patch`?

Reply:
Renamed it to `add_then_patch`